assets/css/main.css

```diff
@@ -246,46 +246,71 @@ html, body {
     list-style: none;
 }
 
-/* index thumbnails */
-.course-thumbnails .advert-thumbnail,
-.course-thumbnails .course-thumbnail {
-    height: 330px;
+
+/* index frontpage */
+
+.frontpage-panel {
+    margin: 3%;
```

Comment:
This looks a bit strange to me. Is the extra space what you were aiming for?

![Screenshot 2020-06-11 at 12 17 24](https://user-images.githubusercontent.com/6674046/84368060-c20d6000-abdd-11ea-8c89-2b4f45e6f063.png)

Reply:
Yes it was, to create a sub-menu-kind-of-feeling :D But maybe we could ask someone to be a tiebreaker and go with that :)